### PR TITLE
CPUB-274 DVP Data API doc improvements

### PR DESCRIPTION
**Pull request by chrispatrick**
### Proposed changes

API documentation improvements for DVP Data API

### CODE REVIEW

File: docker-hub/api/dvp.yaml [hunk 1]
```diff
@@ -6,7 +6,36 @@ info:
     url: https://docs.docker.com/images/logo-docker-main.png
     href: /reference
   description: |
-     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces
+     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:9]:
```suggestion
     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces. Analytics data can be retrieved as raw data, or in a summary format.
```

File: docker-hub/api/dvp.yaml [hunk 2]
```diff
@@ -6,7 +6,36 @@ info:
     url: https://docs.docker.com/images/logo-docker-main.png
     href: /reference
   description: |
-     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces
+     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces  
+       
+     #### Summary data
+
+     In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month).
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:13]:
```suggestion
     In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). 
```

File: docker-hub/api/dvp.yaml [hunk 3]
```diff
@@ -6,7 +6,36 @@ info:
...
+
+     - For every namespace, repo, and reference (tag or digest):
+       - Unique IP address count
+       - Pulls by tag count
+       - Pulls by digest count
+       - Version Check count
+
+     #### Raw data
+
+     In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). Please note that each Action is represented as a single row.
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:23]: I would not capitalize "Action"

```suggestion
     In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). **Note:** each action is represented as a single row.
```

File: docker-hub/api/dvp.yaml [hunk 4]
```diff
@@ -6,7 +6,36 @@ info:
...
+
+     - Type (industry)
+     - Host (cloud provider)
+     - Country (geolocation)
+     - Timestamp
+     - Namespace
+     - Repo
+     - Reference (digest will always be included, we provide tag when available)
+     - HTTP Method
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:32]:
```suggestion
     - HTTP request method
```

File: docker-hub/api/dvp.yaml [hunk 5]
```diff
@@ -6,7 +6,36 @@ info:
...
+
+     In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). Please note that each Action is represented as a single row.
+
+     - Type (industry)
+     - Host (cloud provider)
+     - Country (geolocation)
+     - Timestamp
+     - Namespace
+     - Repo
+     - Reference (digest will always be included, we provide tag when available)
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:31]:
```suggestion
     - Reference (digest is always included, tag is provided when available)
```

File: docker-hub/api/dvp.yaml [hunk 6]
```diff
@@ -6,7 +6,36 @@ info:
...
+     - HTTP Method
+     - Action - can filter on the below
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:33]: Is this like an enum?

File: docker-hub/api/dvp.yaml [hunk 7]
```diff
@@ -53,6 +87,84 @@ x-tagGroups:
...
+    servers:
+      - url: https://hub.docker.com
+    post:
+      security: []
+      tags:
+        - authentication
+      summary: Second factor authentication.
+      operationId: PostUsers2FALogin
+      description: |
+        When user has 2FA enabled, this is the second call to perform after
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:138]:
```suggestion
        When a user has 2FA enabled, this is the second call to perform after
```

File: docker-hub/api/dvp.yaml [hunk 8]
```diff
@@ -6,7 +6,36 @@ info:
...
     href: /reference
   description: |
-     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces
+     The Docker DVP Data API allows [Docker Verified Publishers](https://docs.docker.com/docker-hub/publish/) to view image pull analytics data for their namespaces  
+       
+     #### Summary data
+
+     In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday to Monday) or for a complete month (available on the 1st day of the following month). 
+
+     - For every namespace, repo, and reference (tag or digest):
```
Comment (dvdksn) [inline docker-hub/api/dvp.yaml:15]:
```suggestion
     - For every namespace, repository, and reference (tag or digest):
```

File: docker-hub/api/dvp.yaml [hunk 9]
```diff
@@ -6,7 +6,36 @@ info:
...
+       - Unique IP address count
+       - Pulls by tag count
+       - Pulls by digest count
+       - Version Check count
```
Comment (craig-osterhout) [inline docker-hub/api/dvp.yaml:19]:
```suggestion
       - Version check count
```

File: docker-hub/api/dvp.yaml [hunk 10]
```diff
@@ -6,7 +6,36 @@ info:
...
+     - Timestamp
+     - Namespace
+     - Repo
+     - Reference (digest will always be included, we provide tag when available)
+     - HTTP Method
+     - Action - can filter on the below
+       - Pull by tag
+       - Pull by digest
+       - Version check
+     - User-Agent Tool
```
[the remark this answers is not in the document]
Comment (chrispatrick) [reply docker-hub/api/dvp.yaml:37]: Yep, pretty much

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Pull request merged**